src/InfluxDB.LineProtocol/LineProtocolWriter.cs

```diff
@@ -0,0 +1,239 @@
+﻿using System;
+using System.Globalization;
+using System.IO;
+
+namespace InfluxDB.LineProtocol
+{
+    public class LineProtocolWriter
+    {
+        private static readonly DateTime UnixEpoch = new DateTime(1970, 1, 1, 0, 0, 0, DateTimeKind.Utc);
+
+        private readonly TextWriter textWriter;
```

Comment:
Sounds like a good suggestion; as a private field I think we can consider it for a future change. Going to get this one to `dev`. Thanks for the comments, Thomas 👍